content/pages/resources.md

```diff
@@ -7,7 +7,7 @@ template: resources
 
 * [Solr Quick Start](/guide/solr-tutorial.html)
 
-* [Solr on Docker](https://github.com/docker-solr/docker-solr#getting-started-with-the-docker-image)
+* [Solr ](https://github.com/apache/solr) [on Docker](https://github.com/apache/solr-docker) [(old Documentation)](https://github.com/docker-solr/docker-solr#getting-started-with-the-docker-image)
```

Comment:
I think it is better to link to the reference guide, and not mention old docs...
```suggestion
* [Solr on Docker](https://solr.apache.org/guide/solr/latest/deployment-guide/solr-in-docker.html)